toolkits/global/packages/global-corporate-footer/demo/context.json

```diff
@@ -1,7 +1,65 @@
 {
-	"image": {
-		"src": "../img/logo-springernature_white.svg",
-		"alt": "Springer Nature"
+	"default": {
+		"navigation": {
+			"links": [
+				{
+					"text": "Privacy policy",
+					"url": "/url/to/your/privacy/policy"
+				},
+				{
+					"text": "Manage cookies / Do not sell my data",
+					"url": "/url/to/your/cookie/management/policy"
+				},
+				{
+					"text": "Accessibility statement",
+					"url": "https://www.springernature.com/gp/info/accessibility"
+				}
+			]
+		},
+		"image": {
+			"src": "../img/logo-springernature_white.svg",
+			"alt": "Springer Nature",
+			"link": "https://www.springernature.com/"
+		},
+		"currentYear": 2022
 	},
-	"copyright": "&copy; 2020 Springer Nature Limited"
+	"extended": {
+		"navigation": {
+			"links": [
+				{
+					"text": "[Terms]",
+					"url": "/url/to/your/terms"
+				},
+				{
+					"text": "Privacy policy",
+					"url": "/url/to/your/privacy/policy"
+				},
+				{
+					"text": "California privacy statement (Optional)",
+					"url": "https://www.springernature.com/gp/legal/ccpa"
+				},
+				{
+					"text": "Manage cookies / Do not sell my data",
+					"buttonProperties": "data-cc-action=\"preferences\""
```

Comment:
Should this format also be used for the `default` above? This should always be a button instead of a link